ProductofArrayExceptSelf.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 22 10:26:21 2019
+
+@author: tanvirkaur
+"""
+
+#time complexity = O(n)
+#space complexity = O(n)
+# Leetcode Acceptance = No
+#error in line 17 : index out of range 
+class Solution(object):
+    def productExceptSelf(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: List[int]
+        """
+        res = []*len(nums)
```

Comment:
fix the list
res = [0 for x in range(len(nums))]
I think it should be like this